NEWS.md

```diff
@@ -22,7 +22,8 @@
   ([#2496](https://github.com/JuliaData/DataFrames.jl/pull/2496))
 * `names` now allows passing a predicate as a column selector
   ([#2417](https://github.com/JuliaData/DataFrames.jl/pull/2417))
-
+* added support for casting `AbstractDataFrame` to `Dict` datatype
+  ([#2624](https://github.com/JuliaData/DataFrames.jl/pull/2624))
```

Comment:
```suggestion
  ([#2624](https://github.com/JuliaData/DataFrames.jl/pull/2624))
  
```

Reply:
if you contributed to DataFrames.jl in the future you can consider allowing maintainers to commit suggestions in your PRs (there is such an option when you make a PR) - then it would be easier to quickly fix layout issues.